_api-reference/profile.md

```diff
@@ -236,7 +244,10 @@ Field | Data type | Description
 :--- | :--- | :---
 `type` | String | The Lucene query type into which the search query was rewritten. Corresponds to the Lucene class name (which often has the same name in OpenSearch).
 `description` | String | Contains a Lucene explanation of the query. Helps differentiate queries with the same type.
-`time_in_nanos` | Long | The amount of time the query took to execute, in nanoseconds. In a parent query, the time is inclusive of the execution times of all the child queries.
+`time_in_nanos`	| Long | The total elapsed time for this query, in nanoseconds. For concurrent segment search, `time_in_nanos` is the total time spent across all the slices (the difference between the last completed slice execution end time and the first slice execution start time).
+|`max_slice_time_in_nanos`	| Long | The maximum amount of time taken by any slice to run a query, in nanoseconds.	
+|`min_slice_time_in_nanos`	| Long | The minimum amount of time taken by any slice to run a query, in nanoseconds.	
+|`avg_slice_time_in_nanos`	| Long | The average amount of time taken by any slice to run a query, in nanoseconds.	
```

Comment:
here and below we are talking about min/max/avg which are concurrent search specific fields but the shared example response is for non-concurrent path only. Probably we should share a sample response for concurrent search case as well.

Reply:
https://github.com/opensearch-project/documentation-website/pull/6356/files#r1481845307